### ISSUE TITLE
Fix save data function so it isnt saving raw data

### ISSUE THREAD

**Issue by hokiespurs** [FRF-Remote-Sensing/DuneLidar#58]
We dont want to save the raw data because it takes up too much space.  We should implement the workflow that allows raw data to be regenerated in a minute or so if need be.

**Comment by KateBrodie** [comment]
i agree but for the experiment i had people wanting raw data from each hour so i figured it was easier to save it then have to have then get out code loaded on their system...so after the experiment is over we can change it?
